lib/growthbook/conditions.rb

```diff
@@ -164,9 +162,10 @@ def self.eval_operator_condition(operator, attribute_value, condition_value)
       end
     end
 
-    def self.is_in?(actual, expected)
-      return expected.include?(actual) unless actual.is_a? Array
-        
+    def self.in?(actual, expected)
```

Comment:
I think we may be missing a type definition in the .rbs file for this.

Reply:
Done https://github.com/growthbook/growthbook-ruby/pull/28/commits/eaf38427f39d2991fef2802ec6fb6eb60d214a82, @tinahollygb 😸